### ISSUE TITLE
Weight

### ISSUE THREAD

**Issue by RedAnon** [freiszel/Nema14-dual-drive-geared-extruder#1]
Hi,
how much does the whole contraption weights without a stepper motor?

**Comment by freiszel** [comment]
Hi:

Without motor: 135 g. With motor: 245 g

..

Em qua., 6 de mai. de 2020 às 15:33, Ladislav Ezr jr. <
notifications@github.com> escreveu:

**Comment by RedAnon** [comment]
That's hella light. Have you ever thought about a direct drive version? That would be real sweet for a delta.

**Comment by freiszel** [comment]
Yes, indeed, a direct drive is lighter. I devised the dual drive just to
investigate the increase in extruding force.

..

Em quarta-feira, 6 de maio de 2020, Ladislav Ezr jr. <
notifications@github.com> escreveu:

**Comment by RedAnon** [comment]
Sorry, that's not what I've meant. I was talking about your design but instead of the stepper motor being mounted on the extruder there would be a drive cable. You could also use lighter material (carbon fiber instead of aluminum) to further lower the weight.
That would be the best possible design for a delta printer IMHO.

**Comment by RedAnon** [comment]
Also do you have the specs on the stepper motor you have used? What was the holding torque?

**Comment by freiszel** [comment]
The motor was taken from an old table scanner. In the photo you can see a
tag with few specs: current per phase 0.5 A and phase resistance 3.8 ohms.
No clue about holding torque. The motor is made by Applied Motion, model
5014-819. I have looked on the internet for more detailed specs without
success. It seens to be a very old model.
..


Em qui., 7 de mai. de 2020 às 13:37, Ladislav Ezr jr. <
notifications@github.com> escreveu:

**Comment by RedAnon** [comment]
Ah, that's too bad. Anyway I will try to replicate it over the weekend. Looking at the stepper motors the Sanyo Denki SS2421-5041/SS2422-5041 (0.083N⋅m/0.186N⋅m and 70g/140g) models look real nice. Light and compact. But they are NEMA 17. There doesn't seem to be much choice in the NEMA 14/NEMA 11 (and smaller) sizes when weight is an issue.

How about the gearing? Would you stay with the 4.6:1 gear ratio? Or go higher? 5:1?

**Comment by freiszel** [comment]
I had plastic gears in my scrap box that yelded this 4.6:1  gear ratio.
That was the only reason I choose this ratio.  A 5:1 ratio would word as
well.


...

Em quinta-feira, 7 de maio de 2020, Ladislav Ezr jr. <
notifications@github.com> escreveu:

**Comment by RedAnon** [comment]
Nice. BTW what kind of nozzle did you use?

**Comment by freiszel** [comment]
A standart 0.4 mm nozzle, M6 screw.

Em sexta-feira, 8 de maio de 2020, Ladislav Ezr jr. <
notifications@github.com> escreveu:

**Comment by RedAnon** [comment]
You reckon it would be faster with a Vulcano extruder from E3D?

**Comment by freiszel** [comment]
By "faster" you mean to assembly a complete extruder as soon as possible,
without having the burden of doing everything ? Of course it is!

But if you mean that with a Volcano hotend you can print faster, I doubt if
there is any advantage over others hot ends.



Em sex., 8 de mai. de 2020 às 13:24, Ladislav Ezr jr. <
notifications@github.com> escreveu:

**Comment by RedAnon** [comment]
I meant print speed. Because in theory the more powerful the heater cartridge is the faster you can print. I have no idea how can longer nozzle effect the speed but apparently it does? Even tho the advertised 30mm3/s seems really high and almost everyone peaks at 20. I kinda don't believe the E3D numbers...

**Comment by freiszel** [comment]
For a 1.75mm filament, 30mm3/s means a flow rate of 12.5 mm/s.  The
extruder I devised can reach a flow rate of almost 10 mm/s, not so far from
12.5mm/s.

So, it is plausible that an extruder can reach 12.5 mm/s. But this depends
on the force to push the filament to the hot end. This force does not
depends on the hot end, but only on the motor/gear ratio.
...

Em sexta-feira, 8 de maio de 2020, Ladislav Ezr jr. <
notifications@github.com> escreveu:
